tests/CMakeLists.txt

```diff
@@ -276,6 +276,14 @@ add_test(NAME testLBFGS
          ${CMAKE_CURRENT_SOURCE_DIR}/LBFGS/lrs.in
          ${CMAKE_CURRENT_SOURCE_DIR}/../potentials)
 
+add_test(NAME ChebyshevMVP
+         COMMAND ${PYTHON_EXECUTABLE} ${CMAKE_CURRENT_SOURCE_DIR}/Chebyshev/test.py
+         ${MPIEXEC} ${MPIEXEC_NUMPROC_FLAG} 9 ${MPIEXEC_PREFLAGS}
```

Comment:
@oseikuffuor1 can't you run it with fewer MPI tasks? It looks like your PR is failing because of that. I typically have been trying to keep the number of MPI tasks to 4 or less.

Reply:
I think that the limit on the testing machines is 8 processes.

Reply:
Thanks, I will try a smaller processor decomposition.